### PR TITLE
build: Release v15.0

### PR DESCRIPTION
**Pull request by rbradford**
The Rust manifiest requires a three part version number however we
continue with out plan to use two digits so the last part will not be
used.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by sameo** [comment]
We're doing 0.14.1 and 15.0 at the same time?

**Comment by rbradford** [comment]
> We're doing 0.14.1 and 15.0 at the same time?

I realised the update for the release notes was missing from the master branch. So I copied it in. I'll only include the v15.0 changes in the tag/release.

File: release-notes.md [hunk 1]
```diff
@@ -124,6 +133,98 @@
...
+* Bo Chen <chen.bo@intel.com>
+* Gaelan Steele <gbs@canishe.com>
+* Jianyong Wu <jianyong.wu@arm.com>
+* Michael Zhao <michael.zhao@arm.com>
+* Muminul Islam <muislam@microsoft.com>
+* Rob Bradford <robert.bradford@intel.com>
+* Sebastien Boeuf <sebastien.boeuf@intel.com>
+* Wei Liu <liuwe@microsoft.com>
+* William Douglas <william.douglas@intel.com>
+
```
Comment (rbradford) [inline release-notes.md:215]: nit: Extra line of whitespace here.

**Comment by rbradford** [comment]
> > We're doing 0.14.1 and 15.0 at the same time?
> 
> I realised the update for the release notes was missing from the master branch. So I copied it in. I'll only include the v15.0 changes in the tag/release body.

And I want that to be in the master branch version: a.) to show we do do point releases and b.) So that I don't have to repeats the bugs fixed in there again.

**Review by sboeuf — APPROVED** [review]

**Review by bryteise — APPROVED** [review]
Nit in commit message, s/out/our/

**Review by sameo — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloud-hypervisor"
-version = "0.14.0"
+version = "15.0.0"
```
Comment (likebreath) [inline Cargo.toml:3]: Question: do we intend to use version number as `15.0.0` or `15.0`? The rest of the article is referring to the latter one.

**Review by likebreath — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (rbradford) [reply Cargo.toml:3]: Sadly Rust requires it be 3 digits. But we're only going to use the first 2.

**Pull request merged**